code/modules/clothing/head/f13head.dm

```diff
@@ -457,30 +449,20 @@
 	armor = list("melee" = 80, "bullet" = 50, "laser" = 50, "energy" = 15, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 95, "acid" = 80, "wound" = 80)
 
 /obj/item/clothing/head/helmet/f13/power_armor/advanced
-	name = "advanced power helmet"
-	desc = "It's an advanced power armor MK1 helmet, typically used by the Enclave. It looks somewhat threatening."
-	icon_state = "advhelmet1"
-	item_state = "advhelmet1"
-	armor = list("melee" = 80, "bullet" = 80, "laser" = 85, "energy" = 35, "bomb" = 72, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 50, "wound" = 90)
+	name = "Enclave power armor helmet"
+	desc = "The Enclave Mark II Powered Combat Armor helmet. It looks somewhat threatening."
+	icon_state = "advanced"
+	item_state = "advanced"
+	slowdown = 0.5
```

Comment:
No, this is a GIANT nerf for something that was meant to be good, not slower than salvaged PA.

Reply:
PA should be slow. Let's make most of these numbers similar.